src/components/inputs/localized-money-input/localized-money-input.visualspec.js

```diff
@@ -1,12 +1,17 @@
 import { percySnapshot } from '@percy/puppeteer';
+import { getDocument, queries } from 'pptr-testing-library';
+
+const { getByLabelText } = queries;
 
 describe('LocalizedMoneyInput', () => {
   beforeAll(async () => {
     await page.goto(`${HOST}/localized-money-input`);
   });
 
   it('Default', async () => {
-    await expect(page).toMatch('minimal');
+    const doc = await getDocument(page);
+    const cadInput = await getByLabelText(doc, 'CAD');
+    expect(cadInput).toBeTruthy();
```

Comment:
What do we test here? That there is a CAD input in the document? Why do we use toBeTruthy?

Reply:
> What do we test here

The idea is the same as `expect(page).toMatch('minimal')`. Just to make sure that the route is being rendered, and that our component is being rendered on screen.

> Why do we use toBeTruthy?

 Do you mean instead of using toBeInTheDocument? We can't use that as it's from `jest-dom`